### PR TITLE
Add back overview

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1359

Bare-bones docs to get overview info back into Astro docs. We can add on these over time, but let's get these merged sooner rather than later.

### CODE REVIEW

File: astro/astro-architecture.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+sidebar_label: 'Architecture'
+title: 'Astro architecture'
+id: astro-architecture
+description: Astro uses a multi-plane structure to help make Airflow more secure and reliable. Use this doc to learn how the control plane and data plane work together on the cloud.
```
Comment (kwcanuck) [inline astro/astro-architecture.md:5]:
```suggestion
description: Astro uses a multi-plane structure to help make Airflow more secure and reliable. Learn how the control plane and data plane work together on the cloud.
```

File: astro/astro-architecture.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+
+Astro uses a multi-plane structure to help make Airflow more secure and reliable. The control plane is entirely managed and hosted by Astronomer, while the data plane can run in either your cloud or Astronomer's cloud. The following diagram outlines how the control plane, data plane, and users are connected to enable Astro.
```
Comment (kwcanuck) [inline astro/astro-architecture.md:9]:
```suggestion
Astro uses a multi-plane structure to help make Airflow more secure and reliable. The control plane is entirely managed and hosted by Astronomer, while the data plane can run in either your cloud or the Astronomer cloud. The following diagram shows how the control plane, data plane, and users are connected to enable Astro.
```

File: astro/features.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+title: 'Astro features'
+id: environment-variables
+description: Set environment variables on Astro to specify Airflow configurations and custom logic.
+---
+
+Astro is a managed software service that offers a next-generation experience for modern data teams running Apache Airflow, the open source industry standard for data orchestration.
+
+Astro boasts a hybrid deployment model founded on a control plane hosted by Astronomer and a data plane that is hosted in your cloud environment. Both are fully managed by Astronomer. This model offers the self-service convenience of a fully managed service while respecting the need to keep data private, secure, and within corporate boundaries. This solution optimizes security and reduces operational overhead.
+
+Beyond architecture, Astro offers a suite of first-class features that make it easy to author, run, and monitor data pipelines.
```
Comment (kwcanuck) [inline astro/features.md:12]:
```suggestion
Beyond architecture, Astro offers a suite of first-class features that make it easy to author, run, and monitor data pipelines.
```
image

File: astro/features.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## Feature list
+
+The key features of Astro include:
```
Comment (kwcanuck) [inline astro/features.md:16]:
```suggestion
The key features of Astro include:
```
Remove this introductory statement. It's not necessary.

File: astro/features.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Worker auto-scaling, powered by the Airflow Celery executor + KEDA. See [Worker queues](configure-worker-queues.md).
```
Comment (kwcanuck) [inline astro/features.md:18]:
```suggestion
- Worker auto-scaling, powered by the Airflow Celery executor and KEDA. See [Worker queues](configure-worker-queues.md).
```

File: astro/features.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+- The ability to run multiple clusters in your organization's network on AWS, GCP, or Azure.
+- Astro Runtime, a collection of Docker images that provides a differentiated data orchestration experience. See [Runtime image architecture](runtime-image-architecture.md).
+- Timely support for the latest major, minor, and patch versions of the Apache Airflow open source project.
+- Support for role-based access control (RBAC) and single sign-on (SSO) for secure user management and authentication. See [Configure an identity provider (IdP)](configure-idp.md) and [User permissions](user-permissions.md).
+- An observability experience that gives you insight into the health and resource consumption of your tasks and pipelines in real time. See [Deployment metrics](deployment-metrics.md) and [Data lineage](data-lineage.md).
```
Comment (kwcanuck) [inline astro/features.md:23]:
```suggestion
- Insight into the health and resource consumption of your tasks and pipelines in real time. See [Deployment metrics](deployment-metrics.md) and [Data lineage](data-lineage.md).
```

File: astro/features.md (hunk 3 above)
Comment (paolaperaza) [reply astro/features.md:12]: Let's stick with "easy" -- I wouldn't use that word in documentation ("follow step 1, it's easy!") but I think in a feature overview of our product it's appropriate to underscore the value we bring to market. And that is.. "making all this stuff easy."

File: astro/features.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
```
Comment (paolaperaza) [inline astro/features.md:23]:
```suggestion
- An observability experience that gives you data lineage across your ecosystem as well as real-time insight into the health and resource consumption of your data pipelines on Astro. See  [Data lineage](data-lineage.md] and [Deployment metrics](deployment-metrics.md).
```

File: sidebarsAstro.js [hunk 8]
```diff
@@ -10,9 +10,13 @@
 module.exports = {
   cloud: [
     {
-      type: 'doc',
+      type: 'category',
       label: 'Overview',
-      id: 'overview'
+      link: { type: 'doc', id: 'overview' },
+      items: [
+        'features',
```
Comment (paolaperaza) [inline sidebarsAstro.js:17]: should we call this doc `astro-features` for consistency?

**Review by paolaperaza — APPROVED** [review]
LGTM @jwitz -- I think we're gonna have to revisit this doc soon more holistically, but this is fine for now. Left a few comments.

File: sidebarsAstro.js (hunk 8 above)
Comment (jwitz) [reply sidebarsAstro.js:17]: It would look off in the URL to have `astro/astro-features` imo

**Pull request merged**